src/index.html

```diff
@@ -1,13 +1,304 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>B&O</title>
+    <link
+      rel="icon"
+      type="image/png"
+      href="./icons/ukraine.png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #region head -->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img
+                src="images/Logo.png"
+                alt="bologo"
+                class="top-bar__logo">
+            </a>
+            <div class="top-bar__icons">
+              <div class="top-bar__callback">
+                <a href="tel:+1234567890" class="icon icon--phone">
+                  <a href="#" class = "top-bar__number">
+                    +1 234 5555-55-55
+                  </a>
```

Comment:
Incorrect phone number. Do you need 2 tag a here?
```suggestion
                <a href="tel:+1234567890" class="icon icon--phone">
                  <a href="#" class = "top-bar__number">
                    +1 234 5555-55-55
                  </a>
```
```suggestion
                <a href="tel:+123455555555" class="icon icon--phone">
                  <a href="#" class="top-bar__number">
                    +1 234 5555-55-55
                  </a>
```

Reply:
done , yes i need both of them , because one is invisiable untill you hover on another